modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -367,16 +367,27 @@ definition {
 	@ignore = "Test Stub"
 	@priority = 3
 	test ShowUnderlineEffectsIsDisabledByDefault {
+		task ("Given home page") {
+			Navigator.openURL();
+		}
 
-		//task ("Given When Then") {
+		task ("When access accessibility menu") {
+			AccessibilityMenu.openAccessibilityMenu();
+		}
 
-		// // Given home page
-		// // When access accessibility menu
-		// // Then show underline effects in links is toggled OFF state
-		// // assert body element does not have "c-prefers-link-underline" class
+		task ("Then show underline effects in links is toggled OFF state") {
+			AssertNotChecked.assertNotCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
 
-		//}
+		task ("assert body element does not have 'c-prefers-link-underline' class") {
```

Comment:
Remember I only want to put Given When Then translated to a task. I provide comments below them as suggested poshi steps to include within the Given When Then task.

Reply:
ic... It was only a comment, it couldn't be a step because it does not carry `Given/When/Then`. Or it is just redundant? Thanks for reviewing!